cdk init presently doesn't npm install

OS: Arch Linux (but I don't think it's specific to that)

```$ ~/.cdk/bin/cdk init --type=app
No --language was provided, but 'app' supports only 'typescript', so defaulting to --language=typescript
Initializing a new git repository...
Applying project template app for typescript
Installing CDK modules from /home/leepa/.cdk/repo/npm...

> uglifyjs-webpack-plugin@0.4.6 postinstall /home/leepa/Code/tcdk/node_modules/jsii/node_modules/uglifyjs-webpack-plugin
> node lib/post_install.js
+ aws-cdk-applet-js@0.6.0
+ aws-cdk-codecommit@0.6.0
+ aws-cdk-assert@0.6.0
+ aws-cdk-cloudformation-diff@0.6.0
+ aws-cdk-cloudtrail@0.6.0
+ aws-cdk-cloudfront@0.6.0
+ aws-cdk-codebuild@0.6.0
+ aws-cdk-codepipeline@0.6.0
+ aws-cdk@0.6.0
+ aws-cdk-acm@0.6.0
+ aws-cdk-cx-api@0.6.0
+ aws-cdk-custom-resources@0.6.0
+ aws-cdk-dynamodb@0.6.0
+ aws-cdk-events@0.6.0
+ aws-cdk-iam@0.6.0
+ aws-cdk-kms@0.6.0
+ aws-cdk-ec2@0.6.0
+ aws-cdk-neptune@0.6.0
+ aws-cdk-lambda@0.6.0
+ aws-cdk-quickstarts@0.6.0
+ aws-cdk-rds@0.6.0
+ aws-cdk-rtv@0.6.0
+ aws-cdk-route53@0.6.0
+ aws-cdk-s3@0.6.0
+ aws-cdk-sns@0.6.0
+ aws-cdk-sqs@0.6.0
+ codemaker@0.4.0-pre
+ aws-cdk-util@0.6.0
+ aws-cdk-toolkit@0.6.0
+ jsii-java-runtime@0.4.0-pre
+ jsii@0.4.0-pre
+ jsii-pacmak@0.4.0-pre
+ jsii-kernel@0.4.0-pre
+ jsii-runtime@0.4.0-pre
+ simple-resource-bundler@0.6.0
+ jsii-spec@0.4.0-pre
+ aws-cdk-docs@0.6.0
+ aws-cdk-resources@0.6.0
added 761 packages from 410 contributors and audited 379 packages in 15.858s
found 0 vulnerabilities

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/jsii/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

Executing npm install...
Command failed: npm install
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! tcdk@0.6.0 prepare: `tslint -p . && tsc && chmod a+x bin/tcdk.js`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the tcdk@0.6.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/leepa/.npm/_logs/2018-06-09T07_57_01_653Z-debug.log
```

Looks like it's running npm install before putting the bin folder in with 'tcdk.ts'.

--- a/packages/aws-cdk-toolkit/.npmignore
+++ b/packages/aws-cdk-toolkit/.npmignore
@@ -1,5 +1,6 @@
 # Don't include original .ts files when doing `npm pack`
 *.ts
+!*.template.ts
 !*.d.ts
 coverage
 .nyc_output

